plone/app/widgets/static/requirejs-unset.js

```diff
@@ -1,4 +1,4 @@
-var _old_define = define;
```

Comment:
when in debug mode I get this error:

<img width="2047" alt="bildschirmfoto 2017-06-23 um 09 22 09" src="https://user-images.githubusercontent.com/511761/27470973-644bff00-57f6-11e7-958c-a6990d3e242b.png">

but when in production mode, the error is gone. how that? strange ...